mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,189 @@
+                "type": "keyword"
+              },
+              "sequencing_center": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      "dynamic": {
+        "properties": {
+          "workflow_name": {
+            "type": "keyword"
+          },
+          "workflow_type": {
+            "type": "keyword"
+          },
+          "analysis_software": {
+            "type": "keyword"
+          }
+        }
+      },
+      "file": {
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          },
+          "md5sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          }
+        }
+      },
+      "index_file": {
+        "properties": {
+          "object_id": {
+            "=type": "keyword"
+          },
+          "format": {
+            "type": "keyword"
+          },
+          "md5_sum": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "size": {
+            "type": "integer"
+          }
+        }
+      },
+      "donor": {
+        "type": "nested",
+        "properties": {
+          "id": {
+            "type": "keyword"
+          },
+          "submitter_donor_id": {
+            "type": "keyword"
+          },
+          "gender": {
+            "type": "keyword"
+          },
+          "specimen": {
+            "type": "nested",
+            "properties": {
+              "id": {
+                "type": "keyword"
+              },
+              "submitter_specimen_id": {
+                "type": "keyword"
+              },
+              "tumour_normal_designation": {
+                "type": "keyword"
+              },
+              "specimen_tissue_source": {
+                "type": "keyword"
+              },
+              "specimen_type": {
+                "type": "keyword"
+              },
+              "samples": {
+                "type": "nested",
+                "properties": {
+                  "id": {
+                    "type": "keyword"
+                  },
+                  "submitted_id": {
+                    "type": "keyword"
+                  },
+                  "sample_type": {
+                    "type": "keyword"
+                  },
+                  "matched_normal_submitter_sample_id": {
+                    "type": "keyword"
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "repositories": {
+        "type": "nested",
+        "properties": {
+          "code": {
+            "type": "keyword"
+          },
+          "name": {
+            "type": "keyword"
+          },
+          "organization": {
+            "type": "keyword"
+          },
+          "country": {
+            "type": "keyword"
+          },
+          "url": {
```

Comment:
currently maestro indexes the url as 3 parts: 
```
    @NonNull
    private String baseUrl;

    @NonNull
    private String dataPath;

    @NonNull
    private String metadataPath;
```
does it need to be 1 field ?

Reply:
@blabadi im confused about what these three parts resolve to - can you explain?

Reply:
We discussed with Rob abd the conclusion was that we only need the base url (https://score.rdpc.cancercollaboratory.org) not the other ones that were include in icgc